sea-orm-macros/src/lib.rs

```diff
@@ -7,6 +7,69 @@ mod attributes;
 mod derives;
 mod util;
 
+/// Create an Entity
+///
+/// ### Usage
+///
+/// ```
+/// use sea_orm::entity::prelude::*;
+///
+/// #[derive(Copy, Clone, Default, Debug, DeriveEntity)]
+/// pub struct Entity;
+///
+/// # impl EntityName for Entity {
+/// #     fn table_name(&self) -> &str {
+/// #         "cake"
+/// #     }
+/// # }
```

Comment:
Btw... for some derive macros it's quite complicated to show its usage, as it requires implementing other traits manually or using other derive macros.

I was trying to make Ferris compile for us by adding some placeholder structs, enums and impls. So you can feel free to revert any changes that I made :)

Reply:
Thanks! That improves the documentation further.